static/js/components/SkipFinancialAidDialog.js

```diff
@@ -3,39 +3,47 @@ import React from 'react';
 import Dialog from 'material-ui/Dialog';
 import Button from 'react-mdl/lib/Button';
 
-const skipActions = (cancel, skip) => (
+import { FETCH_PROCESSING } from '../actions';
+import SpinnerButton from './SpinnerButton';
+
+const skipActions = (cancel, skip, fetchAddStatus, fetchSkipStatus) => (
   <div className="actions">
     <Button
       type='button'
       className="secondary-button cancel-button"
       onClick={cancel}>
       Cancel
     </Button>
-    <Button
+    <SpinnerButton
+      component={Button}
+      spinning={fetchSkipStatus === FETCH_PROCESSING}
+      disabled={fetchAddStatus === FETCH_PROCESSING}
       type='button'
-      className="primary-button save-button"
+      className="primary-button save-button skip-button"
       onClick={skip}>
       Pay Full Price
-    </Button>
+    </SpinnerButton>
```

Comment:
It looks like the `actions` class wasn't doing anything important in this case so I just swapped it with `dialogActions`